client/src/pages/sessions/index.tsx

```diff
@@ -72,14 +61,14 @@ const columns: ColumnDef[] = [
     sortable: true,
   },
   {
-    id: "grade",
+    id: "grade-link",
     label: "Grade",
     minWidth: 0,
     align: "center",
     sortable: false,
   },
   {
-    id: "grade",
+    id: "graderGrade",
```

Comment:
we should pick one style for ids and then be consistent. Is it camelCase or -?

Reply:
The id is what the field is called in mongoose (needs to be that in order to sort by that)